react/components/utils/bytesToSize.ts

```diff
@@ -0,0 +1,10 @@
+export const bytesToSize = (bytes: number) => {
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB']
+
+  if (bytes === 0) return '0 Bytes'
```

Comment:
Should this be translated? In Portuguese I think we use the same term, but not sure if we can guarantee the same for other languages.

Reply:
Oh, good question! I will ask to translation team and confirm.